### ISSUE TITLE
Ada language disabled

### ISSUE THREAD

**Issue by Alex-Gamper** [msys2/MINGW-packages#5508]
Why has the 64bit msys2 now suddenly dropped support for the Ada Language with the latest release of GCC version 9.1 ? This was available under GCC version 8.3.

I understand that their may have been issues in the GCC build procedures for 32bit, but as I
understand it the 64bit version was fine (with the exception of bootstrap vs non bootstrap builds
in 8.2)

Alternatively can you advise on how the Ada community can install MSYS2 GCC version 8.3 
with Ada using pacman

**Comment by mati865** [comment]
See https://github.com/msys2/MINGW-packages/issues/4125

**Comment by Alex-Gamper** [comment]
Hi Mati865

Thanks for the solution to my immediate problem. But this however does not address the issue of Ada
being dropped off from the supported languages that now MSYS2 supports, natively. I am in a position
to be able to build GCC from source thru knowledge gained in trial and error, BUT this expertise is far
from what the average dev team is expected to do to get a Ada compiler up and running. The x64 env
should and MUST not be considered the same as the x86 env, and in my view are totally separate env's.

So in summary, Forcing dev teams to re-build GCC with Ada support manually after installing MSYS2 on
a 64bit env is TOTALY OUTRAGEOUS !!!

**Comment by Alexpux** [comment]
@Alex-Gamper, all this problems with packages firstly is because communities that use MSYS2 work want just to use what we do but not helping solving problems. ADA problem is long story and no one going here to discuss about it. You just go here when we decide something that break your work. 
My POV about Ada and all other packages is the same as earlier - I provide packages that building for both architectures only, while we support 32 bit.
MSYS2 is free time project and supporting all this stuff take very much time.
Keep in mind that I’m only one person who create and upload packages and who start MSYS2 project alone.

**Comment by Alex-Gamper** [comment]
Dear Alexpux
I totally agree that the MSYS2 just want it to work, and that is my point (GCC is a General Compiler
Collection, Including Ada and Fortran) and NOT just C?C++ derivatives. So dropping support for Ada is
against the spirit of the FSF's principals, wrt GCC.

Please don't misunderstand, I believe you have done a great service to the MSYS2 env by providing a
GCC toolset for Win32/64 development. BUT please don't drop the x64 support for Ada (or Fortran). 
Both of these languages/communities are very capable, and can/will support the MSYS2 project with
maintenance of the build procedures if required.

In the case of Ada, If you need additional assistance, their is the Google group Comp.lang.ada that can be very helpful!

**Comment by mingwandroid** [comment]
@Alex-Gamper, sorry, if you're want to bring Ada support back then you must do the work <or pay someone to do the work).

Telling others how to spend *their* free time is completely out of order.

**Comment by mingwandroid** [comment]
And don't bring up such spirit of fsf nonsense, please!

**Comment by oscarfv** [comment]
@Alex-Gamper : Ada is known to be broken on i686 since the release of gcc 8, one year ago. The only ones that tried to fix the problem were C/C++ programmers wishing to unblock the upgrade to the new version of gcc i686 package. All this time the i686 users were forced to stay with gcc 7 just for the benefit of Ada users, but no Ada user cared to fix the problem. We tried to involve the core gcc Ada developers but they took a cursory look and walked away.

I can understand your frustration, but you are complaining to the wrong people. You are mistreating those who privileged your interests inconveniencing others'.

By the way, the G in GCC is not "General", but "GNU", and the "spirit of FSF", whatever it means, for sure does not consist on sacrificing our own free time for your personal benefit. You say where we can go to ask for help instead of being you the one who talks to those communities. Sigh.

**Comment by Alexpux** [comment]
Fixed via https://github.com/msys2/MINGW-packages/commit/7e59389dd7fe748b38356f0fc78f5edeb933aa96

**Comment by optikos** [comment]
So with fix 7e759389 applied, does this mean that #4125 is effectively not moribund and that #4125 is effectively has been fixed despite the discounting of the topic in #4125's commentary and #4125's closure long before the fix?  (I am appalled that a known problem's defect was marked •closed• in #4125 simply because no one at that moment had yet volunteered to do the work.  How in heck is anyone supposed to know to volunteer to the work if the need for the work is obfuscated?)
